lib/cisco_node_utils/fabricpath_global.rb

```diff
@@ -231,8 +224,7 @@ def default_linkup_delay_enable
 
     def loadbalance_algorithm
       algo = config_get('fabricpath', 'loadbalance_algorithm')
-      return default_loadbalance_algorithm if algo.nil?
-      algo.first.strip.downcase
+      algo.downcase
```

Comment:
ok

From: Mike Wiebe <notifications@github.com<mailto:notifications@github.com>>
Reply-To: cisco/cisco-network-node-utils <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Friday, November 13, 2015 at 8:42 AM
To: cisco/cisco-network-node-utils <cisco-network-node-utils@noreply.github.com<mailto:cisco-network-node-utils@noreply.github.com>>
Cc: Cisco Employee <dcherian@cisco.com<mailto:dcherian@cisco.com>>
Subject: Re: [cisco-network-node-utils] adding fabricpath topo and fixed feature_set as a class method (#75)


In lib/cisco_node_utils/fabricpath_global.rb<https://github.com/cisco/cisco-network-node-utils/pull/75#discussion_r44803775>:

> @@ -231,8 +224,7 @@ def default_linkup_delay_enable
>
>      def loadbalance_algorithm
>        algo = config_get('fabricpath', 'loadbalance_algorithm')
> -      return default_loadbalance_algorithm if algo.nil?
> -      algo.first.strip.downcase
> +      algo.downcase


Should be able to just do config_get('fabricpath', 'loadbalance_algorithm').downcase

-
Reply to this email directly or view it on GitHub<https://github.com/cisco/cisco-network-node-utils/pull/75/files#r44803775>.